x-pack/plugins/secops/public/components/page/hosts/uncommon_process_table/index.tsx

```diff
@@ -180,6 +178,45 @@ const getUncommonColumns = (startDate: number): Array<Columns<UncommonProcessesE
     name: i18n.HOSTS,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => <>{node.host != null ? extractHostNames(node.host) : getEmptyValue()}</>,
+    render: ({ node }) => {
+      const hosts: HostEcsFields[] = node.host;
+      const draggables = hosts.map(({ id, name }) => {
+        if (name != null && id != null) {
+          return (
+            <DraggableWrapper
+              key={id}
+              dataProvider={{
+                and: [],
+                enabled: true,
+                id: escapeDataProviderId(`uncommon-process-table-${node._id}-hostName-${name}`),
+                name,
+                excluded: false,
+                kqlQuery: '',
+                queryMatch: {
+                  displayField: 'host.name',
+                  displayValue: name,
+                  field: 'host.id',
+                  value: id,
+                },
+                queryDate: {
+                  from: startDate,
+                  to: Date.now(),
```

Comment:
Probably should have a bug in our backlog soon about these `Date.now()`. I do think that's an issue as you pointed out before. It should be looking at the `endDate` and setting its self to that.

Reply:
That's correct. I plumbed it through on the `Host Summary` component, but most the tables and other components that have draggables still need updated as well. I'll create an issue for this.